api-report/garbage-collector.api.md

```diff
@@ -27,7 +32,7 @@ export class GCDataBuilder implements IGarbageCollectionData {
 }
 
 // @public
-export function getChildNodesGCData(gcData: IGarbageCollectionData): Map<string, IGarbageCollectionData>;
+export function getChildNodesGCDetails(gcDetails: IGarbageCollectionSummaryDetails): Map<string, IGarbageCollectionSummaryDetails>;
```

Comment:
Maybe "unpack" is a better verb? Looking strictly at signature, "get" seems like not the best word to describe what it does.